app/server.py

```diff
@@ -1,5 +1,6 @@
+from __future__ import absolute_import
 from flask import Flask, render_template, request, url_for, abort, Response, make_response
-from scrapers import feedgen
+from .scrapers import feedgen
```

Comment:
Again, due to the various pulls not being on the same page (see #148), these things have diverged way too much. I think you should merge this ASAP and then have other pulls update accordingly (sorry for being redundant) @nikhilrayaprolu  Please review.